pinot-core/src/main/java/org/apache/pinot/core/operator/blocks/RowBlock.java

```diff
@@ -16,18 +16,29 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.core.common;
+package org.apache.pinot.core.operator.blocks;
+
+import java.util.List;
+import org.apache.pinot.common.utils.DataSchema;
+
 
 /**
- *
- * Not used for now, since most of the predicates make use of inverted index.
- * Revisit when we push predicate evaluation up into the operator
+ * The {@code RowBlock} contains a block of rows.
```

Comment:
```suggestion
 * The {@code RowBlock} contains a block of rows and a {@link DataSchema}.
 * each row contains a collection of entries which conformed with the DataSchema.
```

Reply:
Good suggestion. I've removed this class for now to make this PR pure clean up. Will re-introduce it in a separate PR